docs-md/tutorials/embedded-connect.md

```diff
@@ -99,10 +101,9 @@ services:
       KSQL_KSQL_CONNECT_WORKER_CONFIG: "/connect/connect.properties"
       KSQL_CONNECT_GROUP_ID: "ksql-connect-cluster"
       KSQL_CONNECT_BOOTSTRAP_SERVERS: "broker:9092"
-      KSQL_CONNECT_KEY_CONVERTER: "io.confluent.connect.avro.AvroConverter"
-      KSQL_CONNECT_KEY_CONVERTER_SCHEMA_REGISTRY_URL: "http://schema-registry:8081"
-      KSQL_CONNECT_VALUE_CONVERTER: "io.confluent.connect.avro.AvroConverter"
-      KSQL_CONNECT_VALUE_CONVERTER_SCHEMA_REGISTRY_URL: "http://schema-registry:8081"
+      KSQL_CONNECT_KEY_CONVERTER: "org.apache.kafka.connect.json.JsonConverter"
```

Comment:
Should there also be a line for `KSQL_CONNECT_KEY_CONVERTER_SCHEMAS_ENABLE`?

Reply:
actually, now that i think of it, i'll just change the default to be StringConverter